kubernetes/fineractmysql-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    app: fineractmysql
+  name: fineractmysql
+spec:
+  ports:
+  - name: "3306"
+    port: 3306
+    targetPort: 3306
+  selector:
+    app: fineractmysql
+status:
+  loadBalancer: {}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  creationTimestamp: null
+  labels:
+    app: fineractmysql-claim0
+  name: fineractmysql-claim0
+spec:
+  accessModes:
+  - ReadOnlyMany
+  resources:
+    requests:
+      storage: 100Mi
+status: {}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: fineract-server
+  name: fineractmysql
+spec:
+  selector:
+    matchLabels:
+      app: fineract-server
+      tier: fineractmysql
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: fineract-server
+        tier: fineractmysql
+    spec:
+      containers:
+      - env:
+        - name: MYSQL_ROOT_PASSWORD
+          value: mysql
+        image: xurror/fineractmysql:1.0
```

Comment:
this (`image: xurror/fineractmysql:1.0`) is NOK... I've found your https://hub.docker.com/r/xurror/fineractmysql, but we cannot depend on that, this is No Go e.g. from both a security perspective (we have no sources and no idea how you built that container... it could be a anything!), and for future maintenance by others. Why not simply use a standard mysql, or even better mariadb, container?  For example https://hub.docker.com/_/mariadb/ ?

Reply:
Do we really need a mysql deployment?